quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -389,6 +389,10 @@ private Path prepareDistribution() {
             if (!inited || (reCreate || !dPath.toFile().exists())) {
                 FileUtil.deleteDirectory(dPath);
                 ZipUtils.unzip(distFile.toPath(), distRootPath);
+                if (System.getProperty("product") != null) {
+                    // MS SQL Server driver might be excluded if running as a product build
+                    copyProvider(dPath, "com.microsoft.sqlserver", "mssql-jdbc");
+                }
```

Comment:
I think we don't necessarily need to add the driver for all tests but maybe rather tie it to the `@WithDatabase` annotation. But it's not a blocker for me, feel free to merge without it.

Reply:
Created similar issue to fix Oracle as well: https://github.com/keycloak/keycloak/issues/23058
CC: @mabartos

Reply:
Thank you @vmuzikar - I'd like to continue without changing this further at least not for KC 22.0.2. Thanks!